package/libs/libtool/Makefile

```diff
@@ -19,8 +19,6 @@ PKG_LICENSE:=GPL-2.0+
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:gnu:libtool
 
-PKG_BUILD_PARALLEL:=0
-
```

Comment:
here too.

Reply:
Can't reproduce any build issues

Reply:
I don't really care whether you can or not, I think it's just simply out of scope of this PR, and your single point test environment is simply not sufficient.  (Same comment applies to the rest of them)

Reply:
If you don't care why even bother commenting?
I'll do builds locally (it'll take some time) and then I'd suggest throwing it on a _reliable_ buildbot as at least one seems dodgy looking at the faillogs.

Reply:
I just didn't word it well enough, my apologies.  I just don't think a single build that compiled ok is evidence that these packages are safe.  I _very much_ want to see this patch land to turn on parallel builds by default for packages with no preference.  I very strongly believe that changing the setting for packages that DO have a preference is out of scope for _this_ commit, and should be done in a follow up.